src/lib/controllers/artifacts-store/artifacts-store.ts

```diff
@@ -0,0 +1,36 @@
+import { gitURI, localPathURI } from '../../tools';
+import { LocalArchiveStoreAdapter } from './adapters/local';
+
+export interface Release {
+	[key: string]: string;
+}
+
+export class ArtifactsStore {
+	public static async create(archiveStoreURI: string): Promise<ArtifactsStore> {
+		let adapter: LocalArchiveStoreAdapter;
+
+		if (await localPathURI(archiveStoreURI)) {
+			adapter = new LocalArchiveStoreAdapter(archiveStoreURI);
+		} else if (gitURI(archiveStoreURI)) {
```

Comment:
This is a skeleton/placeholder for new functionality that should follow up shortly here ;) (hence the 'Not implemented' message). Otherwise, it wouldn't make sense indeed.

Reply:
Error typing is a good point! Coming up.